app/controllers/datastreams_controller.rb

```diff
@@ -48,17 +48,21 @@ def update
     raise ArgumentError, 'Missing content' if params[:content].blank?
 
     begin
-      # check that the content is well-formed xml
       Nokogiri::XML(params[:content], &:strict)
+      @object.datastreams[params[:id]].content = params[:content] # set the XML to be verbatim as posted
+      @object.save
+      Argo::Indexer.reindex_pid_remotely(@object.pid)
+      msg = 'Datastream was successfully updated'
     rescue Nokogiri::XML::SyntaxError
-      raise ArgumentError, 'XML is not well formed!'
+      # if the the content is well-formed xml, inform the user
+      error_msg = 'The datastream could not be saved due to malformed XML.'
+    rescue StanfordError => e
```

Comment:
why catch then? Can you just let it go?

Reply:
Yeah, you are right - doesn't make much sense.  Removed.